src/code/z_actor.c

```diff
@@ -2046,18 +2058,18 @@ void Actor_UpdateAll(PlayState* play, ActorContext* actorCtx) {
                     refActor->world.pos.z, 0, 0, 0, 1);
     }
 
-    sp80 = &D_80116068[0];
+    freezeCategoryPlayerFlags1 = &sFreezeCategoryPlayerFlags1Masks[0];
```

Comment:
I noticed while reading this that it feels a bit ambiguous as to whether this is flags-per-category or categories for each flag when not read with complete context. I know the array is right next to it but I think we can still be clearer here. Maybe `Mask` can still be included? `categoryFreezeMask`?

Reply:
you mean
`freezeCategoryPlayerFlags1`
-> `categoryFreezeMask`
?

why not
it bothers me the order isn't the same as `sFreezeCategoryPlayerFlags1Masks` though

Reply:
fwiw I think `sCategoryFreezeMasks` is also fine for the data itself. the fact that its player state flags 1 is pretty easily gleaned from usage and the fact that all of the data is already state 1 (And it can also be clarified in a commend above)

Reply:
I'll change it to `freezeCategoryMask`, tell me what you think